notebook/converted/benchmark_results.py

```diff
@@ -1313,3 +1315,143 @@ def return_unlabeled():
 
 
 
+
+# ## Sample Split Positive Control
+
+# In[14]:
+
+
+in_files = glob.glob('../../results/sample-split.*.tsv')
+print(in_files[:5])
+
+
+# In[15]:
+
+
+tissue_metrics = pd.DataFrame()
+
+for path in in_files:
+    new_df = pd.read_csv(path, sep='\t')
+    model_info = path.strip('.tsv').split('biobert.')[-1]
+    model_info = model_info.split('.')
+    model_info = model_info[-1]
+    model_info = model_info.split('_')
+        
+    supervised_model = '_'.join(model_info[:2])
+             
+    new_df['supervised'] = supervised_model
+    
+    new_df['seed'] = model_info[-1]
+            
+    tissue_metrics = pd.concat([tissue_metrics, new_df])
+    
+tissue_metrics['train_count'] = tissue_metrics['train sample count']
+tissue_metrics['supervised'] = tissue_metrics['supervised'].str.replace('pytorch_supervised', 'three_layer_net')
+
+tissue_metrics
+
+
+# In[16]:
+
+
+plot = ggplot(tissue_metrics, aes(x='train_count', y='balanced_accuracy', color='is_pretrained')) 
```

Comment:
Why doesn't pretraining work for logistic regression? Or is it just that the baseline performance is so much lower than for the NN that you can't see any real effect?